cypress/e2e/cloud/usage.test.ts

```diff
@@ -26,7 +27,7 @@ describe('Usage Page Free User No Data', () => {
 
   it('should display the usage page common features', () => {
     // Display the upgrade button when the user is a free user
-    cy.get('.cf-page-header--fluid').children().should('have.length', 2)
+    cy.get('.cf-page-header--fluid').children().should('have.length', 1)
```

Comment:
This change is necessary because the location of the upgrade button has moved. The fluid page-header will always only have one item present in that row.